### PR TITLE
feat: add className exported feature so we can add custom classes to …

### PR DESCRIPTION
**Pull request by 0x090909**
feat: add className exported feature so we can add custom classes to our tab root

### CODE REVIEW

File: src/lib/tabs/Tabs.svelte [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <script lang="ts">
 	import { onMount } from "svelte";
 	import { page } from "$app/stores";
+	import {cn} from "$lib/util/utils.js";
 
 	export let border: boolean = true;
+	export let className: string = "";
```
Comment (ieedan) [inline src/lib/tabs/Tabs.svelte:7]: Change to 
```
let className: string | undefined | null = undefined;
export { className as class}
```

**Review by ieedan — CHANGES_REQUESTED** [review]

File: src/lib/tabs/Tabs.svelte (hunk 1 above)
Comment (0x090909) [reply src/lib/tabs/Tabs.svelte]: I changed

**Comment by ieedan** [comment]
This looks good on my phone I am gonna be home in a half and hour or so and I'll take one more look on my computer

**Comment by 0x090909** [comment]
perfect I wanted to add some padding left and right but without the class prop exported is very hard

File: src/lib/tabs/Tabs.svelte [hunk 2]
```diff
@@ -120,36 +123,35 @@
...
 
-<div
-	data-border={border}
-	class="flex place-items-center data-[border=true]:border-b border-gray-100 overflow-hidden
-dark:border-gray-900 relative group/tabs"
->
+
+<div data-border={border}
+	 class={cn('flex place-items-center overflow-hidden data-[border=true]:border-b border-gray-100 relative group/tabs ', className)}>
+
```
Comment (ieedan) [inline src/lib/tabs/Tabs.svelte:129]: When you copied this is clipped out some of the class names and the correct border color is not being applied in dark mode.
Should change to:
```svelte
class={cn('flex place-items-center data-[border=true]:border-b border-gray-100 overflow-hidden
dark:border-gray-900 relative group/tabs', className)}>
```

Comment (0x090909) [reply src/lib/tabs/Tabs.svelte]: please check

**Comment by ieedan** [comment]
Looks good I'll let it redeploy we will need to bump version in package.json as well.

You can change it to v0.7.13 and then run `npm install` to update package-lock.

**Comment by ieedan** [comment]
Deployment looks good just bump package version and we should be good to go

**Comment by 0x090909** [comment]
Great!! thanks

**Review by ieedan — DISMISSED** [review]

**Comment by ieedan** [comment]
This fixes #265

**Comment by 0x090909** [comment]
sorry I closed by mistake the pr

**Comment by 0x090909** [comment]
> Looks good I'll let it redeploy we will need to bump version in package.json as well.
> 
> You can change it to v0.7.13 and then run `npm install` to update package-lock.

npm says there is no v0.7.13 version

**Comment by ieedan** [comment]
Change the version of this package

**Comment by 0x090909** [comment]
done

**Comment by ieedan** [comment]
Thanks !

**Comment by 0x090909** [comment]
I think you have to create a new tag

**Pull request merged**